sdks/java/core/src/main/java/org/apache/beam/sdk/coders/AvroCoder.java

```diff
@@ -107,10 +107,14 @@
  * org.apache.beam.sdk.transforms.GroupByKey} operations.
  *
  * @param <T> the type of elements handled by this coder
+ * @deprecated Avro classes are deprecated to use in module <code>beam-sdks-java-core</code> and
```

Comment:
Nitpick... maybe like below? Though not important
```suggestion
 * @deprecated Avro related classes are deprecated in module <code>beam-sdks-java-core</code> and
```

Reply:
Please also mention the new import that should be used instead to make the migration as easy as possible. Same for all below...